app/src/main/java/com/example/drawingApp/MainActivity.kt

```diff
@@ -169,6 +172,10 @@ class MainActivity : AppCompatActivity() {
             model.layerListClicked()
         }
 
+        findViewById<LinearLayout>(R.id.tabSheet).let { view ->
```

Comment:
Using let doesn't make sense like this for a one liner that isn't a safe call lol

Reply:
Yea it was there from when I originally made the bottom sheet expand, before I moved it into the util